### DISCUSSION TITLE
Receiving Data as Hex

### DISCUSSION THREAD

**Discussion by iyaiyayohhh** [PhilipsHue/flutter_reactive_ble#863] [Q&A]
Hi! 

I currently have a BLE device that sends a hex value that contains different data that I want to parse for my Flutter application. I would like to ask if this is possible with this package. Any tips on the implementation would be great as I am new to Flutter.

**Comment by Terminal-000** [comment]
Hi
This is possible with every means and I think this package is the best maintained package out there for BLE development. I have an experience of getting real-time ECG Data from a BLE device and Parsing it simultaneously and the code is now at the production level.
As a starter tip I would suggest you to use a state management library such as bloc or provider to manage your Bluetooth device states. I personally use bloc. This will make your life a whole lot easier.
